filter.js

```diff
@@ -0,0 +1,9 @@
+let arr = [5, 3, 8, 1];
```

Comment:
use const for arrays that you will not change
```suggestion
const arr = [5, 3, 8, 1];
```